src/main/java/com/infomaniak/lib/core/extensions/AppCompatActivityExtensions.kt

```diff
@@ -43,3 +44,7 @@ private fun AppCompatActivity.noLocalesAreAccepted(): Boolean {
     }
     return true
 }
+
+fun AppCompatActivity.keepSplashscreenVisibleWhileLoading() {
+    findViewById<View>(android.R.id.content).viewTreeObserver.addOnPreDrawListener { false }
```

Comment:
Do we have to remove the listener at some point with `removeOnPreDrawListener` ?

Reply:
This function is used in the LaunchActivity, to keep the Splashscreen displayed while the Activity is computing stuff (loading mails, etc…).
When the computing is done, the LaunchActivity will finish, so there is no need to remove the listener.